src/main/java/frc/robot/commands/RunClimbWinch.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climb;
+
+public class RunClimbWinch extends Command {
+  /** Creates a new RunClimbWinch. */
+  private Climb climb;
+
+  public RunClimbWinch(Climb climb) {
+    this.climb = climb;
+    addRequirements(climb);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    climb.runWinch();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    climb.stopWinch();
```

Comment:
this doesn't actually have to be here, because the periiodic() in the subsystem is going to stop any time the winch limit is hit.

lets just add comment to that effect hhere.... that we don't actually have to ocall thtis but we're choosing to do so because we really wan tto not run past the end of the wench.